pkg/cli/common.go

```diff
@@ -268,7 +268,7 @@ always:  pull base and SBOM scanner images even if the named images are present
 missing: pull base and SBOM scanner images if the named images are not present in store.
 never:   only use images present in store if available.
 newer:   only pull base and SBOM scanner images when newer images exist on the registry than those in the store.`)
-	fs.Lookup("pull").NoOptDefVal = "true" //allow `--pull ` to be set to `true` as expected.
+	fs.Lookup("pull").NoOptDefVal = "missing" //allow `--pull ` to be set to `missing` as expected.
```

Comment:
That comment is confusing.

Reply:
What would you prefer?

Reply:
Something like 'treat a `--pull` with no argument like `--pull=missing`"